index.html

```diff
@@ -1494,8 +1494,8 @@
 
 <p>
   <span class="practicelab" id="principle-collective-privacy">
-    Groups and various forms of institutions should best protect and support autonomy by making
-    decisions collectively rather than individually to either prevent or enable data
+    Groups and institutions should support autonomy by making
+    decisions collectively to either prevent or enable data
     sharing, and to set defaults for data processing rules.
```

Comment:
I'm not really sure that this says what it should now.  The right words are hard to get here, but there are few different things that you might say:

1. Institutions need to consider individual autonomy as a goal when making decisions about how to structure the systems they produce, whether it be regulation or voluntary standards.
2. Institutions need to consider the potential for individuals to have diverse goals as it relates to autonomy, rather than thinking of individuals as a uniform collective, and account for those diverse goals when making decisions [etc...].
3. In some settings, decisions made collectively for a population can provide the best ability to ensure that individual autonomy is respected, particularly where actions by a single individual might otherwise have stronger adverse consequences for privacy, such as retaliation or becoming more identifiable as a result of that action.
4. Collective decision-making can provide better outcomes where decisions need to carefully balance individual autonomy against collective harms or benefits.

I'm sure that there are more and more nuanced things to be said here, but I don't know which of these applies.  Perhaps all of them and more.

Reply:
These are useful points to make in the longer explanatory text of this section. There are some existing examples for when collective decision-making can facilitate individual autonomy (like when the decisions are too frequent or complex for an individual), but you should suggest some additional conditions.

Maybe we could merge the fix to the principle, and then Martin you could propose either longer text for this section, or some separate piece of writing that we could cite.

Reply:
I don't disagree that there's considerable potential for subtlety and nuance in how institutions relate to individual autonomy. However, I am concerned that in trying to get into too much detail here exceeds the bounds of our legitimate expertise. Doing it right would also extend the text by a lot, and in a way that would not alleviate concerns made by some (including @martinthomson :) that this document is already academic AF.